packages/cli/src/__mocks__/fs.ts

```diff
@@ -41,11 +46,14 @@ class MockStream {
                 message: prevFile.message.concat(message)
             }
         }
+        setTimeout(() => {
```

Comment:
What is the intent here please?
I don't think adding the timer here is going to block the thread, in case that was the expected behavior?

Reply:
I think it should when using `jest.useFakeTimers` https://jestjs.io/docs/en/timer-mocks

Reply:
Isn't the behavior there relying on the `callback` being called?
I haven't used these timers features from Jest so maybe it works as-is, all good if that's the case!

Reply:
Oh you are right. I think I just had it left over from some debugging but it's not needed. Thanks!